examples/spring-boot/src/main/java/io/opentracing/contrib/jaxrs2/example/spring/boot/JerseyConfig.java

```diff
@@ -23,13 +22,9 @@
     @Inject
     public JerseyConfig(Tracer tracer) {
         Client client = ClientBuilder.newClient();
+        client.register(new ClientTracingFeature(new Builder(tracer)));
```

Comment:
Can't this use the same pattern as the server, i.e. `client.register(new ClientTracingFeature.Builder(tracer).build())` - haven't checked the code so possibly this is supported.